app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -174,14 +174,14 @@ object BuildData {
 sealed trait BuildJobType { def asString: String }
 
 object BuildJobType {
-  case object Job         extends BuildJobType { override val asString = "job"         }
-  case object Pipeline    extends BuildJobType { override val asString = "pipeline"    }
-  case object Performance extends BuildJobType { override val asString = "performance" }
+  case object Job         extends BuildJobType { override val asString = "job"          }
+  case object Pipeline    extends BuildJobType { override val asString = "pipeline"     }
+  case object PullRequest extends BuildJobType { override val asString = "pull-request" }
```

Comment:
Can we still distinguish `Performance` tests - I think this would be useful. Is this a sub-type of Job?